sms/v1/openapi.yaml

```diff
@@ -318,270 +492,119 @@ paths:
-                type: string
-              callbackUrl:
-                description: Your URL for delivering the DLRs. Leave empty for no DLR
-                type: string
-              callbackMethod:
-                description: Your prefered HTTP method for the DLR callback. Possible values POST/GET
-                type: string
-              partLimit:
-                description: Up to how many parts you allow this message to be concatenated
-                type: integer
-                format: int32
-                default: 4
-              trate:
-                description: >-
-                  Up to how much you allow the transcoder to replace original
-                  characters to make it GSM7 compatible in order for it to fit
-                  in less parts. Possible decimal values from 0 to 1. (0.5 would
-                  mean 50% or the original content can be replaced)
-                type: number
-                format: double
-                default: 0
-              mrate:
-                description: >-
-                  Up to how much you allow the transcoder to skip original
-                  characters to make it GSM7 compatible. Skipped character will
-                  be replaced with '.'. Possible decimal values from 0 to 1.
-                  (0.5 would mean tht 50% of the original message can be skipped
-                  in order to be fitting to less parts)
-                type: number
-                format: double
-                default: 0
-              upperCased:
-                description: >-
-                  If you allow the transcoder to try an uppercased version of
-                  the content in case this improves the produced parts of the
-                  message
-                type: boolean
-                default: true
-              keepEmojis:
-                description: >-
-                  If transcoder is not allowed to replace emojis with a text translation.
-                  By default the transcoder will not be allowed to replace them, thus text
-                  will result to UNICODE.
-                type: boolean
-                default: true
-              flash:
-                description: >-
-                  If this sms will be delivered as flash. Some networks do not
-                  support his.
-                type: boolean
-                default: false
-            required:
-              - to
-              - from
-              - message
   securitySchemes:
-    APIKeyHeader:
+    ApiKeyAuth:
       type: apiKey
       in: header
-      name: apiKey
+      name: apikey
+      description: |
+        The API key used for this operation. We check as well that the API key has the phone number
+        (either on sending messages or interacting with other phone number based features) assigned.
   schemas:
     RestMessagePart:
       type: object
+      description: The message you would like to send
```

Comment:
I think it's not about the "message you would like to send" but rather about the parts the message request is split into. @0xdeadbee Am I right?

Reply:
i wrote something... still i don't like it very much, but it closer to what this field is